### PR TITLE
Common - Add cache to `fnc_isModLoaded`

### PR DESCRIPTION
**Pull request by Salluci**
**When merged this pull request will:**
- Title.
- Move remaining manual CfgPatches lookups to use the function.
- Change a CfgVehicles >> typeOf lookup to configOf.
- Change overpressure damage to only show blood effect if damage is enabled.

### IMPORTANT

- [ ] If the contribution affects [the documentation](https://github.com/acemod/ACE3/tree/master/docs), please include your changes in this pull request so the documentation will appear on the [website](https://ace3mod.com/).
- [x] [Development Guidelines](https://ace3mod.com/wiki/development/) are read, understood and applied.
- [x] Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

**Comment by jonpas** [comment]
Is it actually faster? Config reading is pretty fast to begin with, and this only does one config read. Benchmarks would be nice.

**Comment by Salluci** [comment]
Old:
```
Result:
0.0178 ms

Cycles:
10000/10000

Code:
["ACE_Medical"] call ace_common_fnc_isModLoaded
```

PR:
```
Result:
0.0068 ms

Cycles:
10000/10000

Code:
["ACE_Medical"] call isModLoadedCached
```

File: addons/common/functions/fnc_isModLoaded.sqf [hunk 1]
```diff
@@ -1,10 +1,10 @@
 #include "script_component.hpp"
 /*
  * Author: Glowbal
```
Comment (PabstMirror) [inline addons/common/functions/fnc_isModLoaded.sqf:3]:
```suggestion
 * Author: Glowbal, Salluci
```

**Review by PabstMirror — APPROVED** [review]

File: addons/common/functions/fnc_isModLoaded.sqf (hunk 1 above)
Comment (Salluci) [reply addons/common/functions/fnc_isModLoaded.sqf:3]: Ooooo I get my name on a thing, shiny!

**Pull request merged**